packages/gestalt/src/ModuleExpandable.js

```diff
@@ -36,31 +34,34 @@ export default function ModuleExpandable({
 
   return (
     <Box borderStyle="shadow" rounding={4}>
-      {items.map(({ children, icon, iconAccessibilityLabel, summary, title, type }, index) => (
-        <Fragment key={index}>
-          <ModuleExpandableItem
-            accessibilityCollapseLabel={accessibilityCollapseLabel}
-            accessibilityExpandLabel={accessibilityExpandLabel}
-            icon={icon}
-            iconAccessibilityLabel={iconAccessibilityLabel}
-            id={`${id}-${index}`}
-            isCollapsed={expandedId !== index}
-            onModuleClicked={(isExpanded) => {
-              if (onExpandedChange) {
-                onExpandedChange(isExpanded ? null : index);
-              }
-              setExpandedId(isExpanded ? null : index);
-            }}
-            summary={summary}
-            title={title}
-            type={type}
-          >
-            {children}
-          </ModuleExpandableItem>
+      {items.map(
+        ({ badgeText, children, icon, iconAccessibilityLabel, summary, title, type }, index) => (
+          <Fragment key={index}>
+            {index > 0 && <Divider />}
```

Comment:
this is the same logic, just simpler (and more functional, no need to reference something outside the iterative loop)